src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Raw.cs

```diff
@@ -194,9 +263,35 @@ private void WriteRawValueCore(ReadOnlySpan<byte> utf8Json, bool skipInputValida
             }
 
             utf8Json.CopyTo(output.Slice(BytesPending));
-            BytesPending += len;
+            BytesPending += (int)len;
 
             SetFlagToAddListSeparatorBeforeNextItem();
         }
+
+        private void GrowIfNeeded(int len)
```

Comment:
Why is this added method necessary?

Reply:
Removed.
Added previously because I cannot figure out better way to reduce duplicated code between `ReadOnlySpan `and `ReadOnlySequence `overloads, so just make them shorter in main methods.